App/src/main/java/com/gluonhq/chat/views/ChannelPresenter.java

```diff
@@ -60,6 +63,16 @@ public void updateItem(Channel item, boolean empty) {
                 }
             }
         });
+        notificationPane.setText("Update Available");
+        notificationPane.getActions().addAll(new Action("Install", ae -> {
+            service.installNewVersion();
+            notificationPane.hide();
+        }));
+        service.newVersionAvailable().addListener((o, ov, nv) -> {
```

Comment:
Shouldn't we do the check (also) in the HomePresenter?

Reply:
Yes, but if a new version is needed since there is a change in how channels are loaded, we will never reach the ChannelPresenter,